scripts/day2/edge-release-images.txt

```diff
@@ -1,38 +1,40 @@
-edge/akri-agent:v0.12.20
-edge/akri-controller:v0.12.20
-edge/akri-debug-echo-discovery-handler:v0.12.20
-edge/akri-onvif-discovery-handler:v0.12.20
-edge/akri-opcua-discovery-handler:v0.12.20
-edge/akri-udev-discovery-handler:v0.12.20
-edge/akri-webhook-configuration:v0.12.20
-edge/baremetal-operator:0.5.1
-edge/cluster-api-controller:v1.6.2
-edge/ip-address-manager:1.6.0
-edge/cluster-api-provider-metal3:v1.6.0
-edge/cluster-api-provider-rke2-bootstrap:0.4.1
-edge/cluster-api-provider-rke2-controlplane:0.4.1
-edge/edge-image-builder:1.0.1
-edge/endpoint-copier-operator:v0.2.0
-edge/frr:8.4
-edge/frr-k8s:v0.0.8
-edge/ip-address-manager:v1.6.0
-edge/ironic:23.0.2.1
-edge/ironic-ipa-downloader:1.3.4
-edge/kube-rbac-proxy:v0.14.2
+edge/3.1/akri-agent:v0.12.20
```

Comment:
Should Elemental, Rancher, NeuVector be part of this list?

Reply:
Rancher, Elemental and NeuVector images are shipped with the `rancher-images.txt` file that comes with each rancher [release](https://github.com/rancher/rancher/releases).